Modules/DataModel/Sources/PocketCastsDataModel/Public/Model/BaseEpisode.swift

```diff
@@ -57,4 +57,5 @@ import Foundation
     func mayContainChapters() -> Bool
 
     var hasBookmarks: Bool { get }
+    var isStub: Bool { get set }
```

Comment:
It would be good to add a comment here:

```suggestion
    /// This property is true when...
    var isStub: Bool { get set }
```